### PR TITLE
BEPs redirect cleanup

### PR DESCRIPTION
**Pull request by iamdamion**
- Cleaned up BEPs links to not redirect.
- Made all external links in new tab.
- Adjusted beps table jinja file to take link from yaml file

### CODE REVIEW

File: data/beps.yml [hunk 1]
```diff
@@ -132,12 +140,13 @@
 
 -   number: '022'
     title: Magnetic Resonance Spectroscopy (MRS)
+    link: https://docs.google.com/document/d/1pWCb02YNv5W-UZZja24fZrdXLm4X7knXMiZI7E2z7mY/
```
Comment (Remi-Gau) [inline data/beps.yml:143]: I believe there is a PR for this now

File: data/beps.yml [hunk 2]
```diff
@@ -247,11 +262,12 @@
 
 -   number: '034'
     title: Computational modeling
+    link: https://docs.google.com/document/d/1NT1ERdL41oz3NibIFRyVQ2iR8xH-dKY-lRCB4eyVeRo/
```
Comment (Remi-Gau) [inline data/beps.yml:265]: I believe there is a PR for this on but am not 100% sure

File: data/beps_completed.yml [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 #   title:
+#   link:
 #   leads:
 #     - name:
 #   content:
 #     - raw
 
 -   number: '001'
     title: Quantitative MRI (qMRI)
+    link: https://github.com/bids-bep001/bids-specification
```
Comment (Remi-Gau) [inline data/beps_completed.yml:13]: probably better to point to the relevant appendix section of the spec website

File: data/beps_completed.yml [hunk 4]
```diff
@@ -20,19 +22,21 @@
 
 -   number: '002'
     title: BIDS Models Specification
+    link: https://docs.google.com/document/d/1bq5eNDHTb6Nkx3WUiOBgKvLNnaa5OMcGtD0AZ9yms2M/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:25]: probably better to point to URL of the current bids stats model website

this website will ultimately move when we merge all our website but let's start with valid URL for now

File: data/beps_completed.yml [hunk 5]
```diff
@@ -20,19 +22,21 @@
...
-    outcome: Became an [independent specification with validator](https://bids-standard.github.io/stats-models/) to execute statistical models, with minimal
+    outcome: Became an [independent specification with validator](https://bids-standard.github.io/stats-models/){:target="_blank"} to execute statistical models, with minimal
         configuration and intervention required from the user.
     google_doc_created: 2016-09
     pull_request_created: 2018-10
     pull_request_merged: 2023-08
 
 -   number: '003'
     title: Common Derivatives
+    link: https://github.com/bids-standard/bids-specification/pull/265
```
Comment (Remi-Gau) [inline data/beps_completed.yml:39]: probably best to point to the relevant derivtives section of the specification

File: data/beps_completed.yml [hunk 6]
```diff
@@ -45,6 +49,7 @@
 
 -   number: '005'
     title: Arterial Spin Labeling (ASL)
+    link: https://docs.google.com/document/d/15tnn5F10KpgHypaQJNNGiNKsni9035GtDqJzWqkkP6c/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:52]: link to the ASL section in the spec

File: data/beps_completed.yml [hunk 7]
```diff
@@ -58,6 +63,7 @@
 
 -   number: '006'
     title: Electroencephalography (EEG)
+    link: https://docs.google.com/document/d/1ArMZ9Y_quTKXC-jNXZksnedK2VHHoKP3HCeO5HPcgLE/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:66]: link to the EEG section in the spec

File: data/beps_completed.yml [hunk 8]
```diff
@@ -70,6 +76,7 @@
 
 -   number: '007'
     title: Hierarchical Event Descriptor (HED) Tags
+    link: https://docs.google.com/document/d/1gs7USIVO1OApZs3IVo6TeXw9_tfzxhjM2-m--ex4g6o/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:79]: link to the HED section in the spec

File: data/beps_completed.yml [hunk 9]
```diff
@@ -80,6 +87,7 @@
 
 -   number: '008'
     title: Magnetoencephalography (MEG)
+    link: https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:90]: link to the MEG section in the spec

File: data/beps_completed.yml [hunk 10]
```diff
@@ -90,6 +98,7 @@
 
 -   number: '009'
     title: Positron Emission Tomography (PET)
+    link: https://github.com/bids-standard/bids-specification/pull/633
```
Comment (Remi-Gau) [inline data/beps_completed.yml:101]: link to the PET section in the spec

File: data/beps_completed.yml [hunk 11]
```diff
@@ -100,6 +109,7 @@
 
 -   number: '010'
     title: intracranial Electroencephalography (iEEG)
+    link: https://docs.google.com/document/d/1qMUkoaXzRMlJuOcfTYNr3fTsrl4SewWjffjMD5Ew6GY/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:112]: link to the iEEG section in the spec

File: data/beps_completed.yml [hunk 12]
```diff
@@ -111,6 +121,7 @@
 
 -   number: '018'
     title: Genetic information
+    link: https://docs.google.com/document/d/1uRkgyzESLKuGjXi98Z97Wh6vt-iLN5nOAb9TG16CjUs/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:124]: link to the genetics section in the spec

File: data/beps_completed.yml [hunk 13]
```diff
@@ -123,6 +134,7 @@
 
 -   number: '027'
     title: BIDS Applications 2.0
+    link: https://docs.google.com/document/d/104HLZedFtx0TaXEUwd7eyWvJUlc0CcSUtCzwjNgmGxE/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:137]: link to the bids-execution website

this one too will be merged with the rest but once again let's use valid URLs for now

File: data/beps_completed.yml [hunk 14]
```diff
@@ -134,6 +146,7 @@
 
 -   number: '029'
     title: Virtual and physical motion data
+    link: https://docs.google.com/document/d/1iaaLKgWjK5pcISD1MVxHKexB3PZWfE2aAC5HF_pCZWo/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:149]: link to motion part of the spec

File: data/beps_completed.yml [hunk 15]
```diff
@@ -146,6 +159,7 @@
 
 -   number: '030'
     title: Near Infrared Spectroscopy (NIRS)
+    link: https://docs.google.com/document/d/1FzPt7TVpIAtlhrF9_9b2VBCZOrRXJASEMyHndhDQKl8/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:162]: link to NIRS seection of the spec

File: data/beps_completed.yml [hunk 16]
```diff
@@ -157,6 +171,7 @@
 
 -   number: '031'
     title: Microscopy
+    link: https://docs.google.com/document/d/1Nlu6QVQrbOQtdtcRarsONbX5SrOubXWBvkV37LRRqrc/
```
Comment (Remi-Gau) [inline data/beps_completed.yml:174]: link to microscopy section of the spec

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
mostly needs changing the links of the merged BEPs to point to either real websites or the relevant section of the spec

**Comment by iamdamion** [comment]
updated links according to notes above

**Review by Remi-Gau — APPROVED** [review]
Awesome! Thanks!

**Pull request merged**